apps/vs-code-intellisense/.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "root": true,
```

Comment:
Question: should this use Paste's global eslint config? These rules don't match what we have in the global one.